python/ray/serve/controller.py

```diff
@@ -846,13 +891,17 @@ async def update_backend_config(
             elif isinstance(config_options, dict):
                 update_data = config_options
 
-            stored_backend_config = self.backends[backend_tag].backend_config
+            stored_backend_config = self.configuration_store.backends[
+                backend_tag].backend_config
             backend_config = stored_backend_config.copy(update=update_data)
             backend_config._validate_complete()
-            self.backends[backend_tag].backend_config = backend_config
+            self.configuration_store.backends[
```

Comment:
`configuration_store.backends[backend_tag].backend_config` looks like a common pattern in controller, let's create getter and setter? `store.get_backend_config(tag), ...`